### DISCUSSION TITLE
Map type of ContactResultMap

### DISCUSSION THREAD

**Discussion by rjoomen** [tesseract-robotics/tesseract#991] [Q&A] [answered]
ContactResultMap says in the comments it's an unordered_map [here](https://github.com/tesseract-robotics/tesseract/blob/master/tesseract_collision/core/include/tesseract_collision/core/types.h#L141), but it actually uses an aligned [normal map](https://github.com/tesseract-robotics/tesseract/blob/master/tesseract_collision/core/include/tesseract_collision/core/types.h#L161).

Is the actual map type incorrect, or the comments?

**Comment by Levi-Armstrong** [comment]
The comment is incorrect.

**Comment by rjoomen** [comment, marked as answer]
Fixed in #1000
